stubs/inertia/resources/js/Pages/Auth/ResetPassword.vue

```diff
@@ -68,10 +68,7 @@
         methods: {
             submit() {
                 this.form.post(this.route('password.update'), {
-                    onSuccess: () => {
-                        this.form.password = ''
-                        this.form.password_confirmation = ''
-                    }
+                    onFinish: () => this.form.reset('password', 'password_confirmation'),
```

Comment:
Shouldn't this only happen on success?

Reply:
That'd change the behaviour, as with the 'old' version of Inertia, `onSuccess` was also triggered on validation errors etc.. This way, the behaviour stays the same, all while retaining the benefits I mentioned in my earlier comment